articles/Third Party Tools and Libraries/pdf-printing.md

```diff
@@ -0,0 +1,48 @@
+<!--
+parent: articles
+created_at: '2017-05-09 09:25:30'
+updated_at: '2017-05-09 09:25:30'
+authors:
+    - 'Jean-Sébastien Conan'
+tags:
+    - 'Backend'
+    - 'Tools'
+    - 'taoBooklet'
+    - 'PHP'
+    - 'PDF'
+    - 'wkhtmltopdf'
+    - 'wkhtml'
+    - 'Print'
+    - 'Document'
+-->
+
+Install wkHTMLtoPDF
+====================
+
+![wkhtmltopdf logo](../resources/third-party/wkhtmltopdf.png) https://wkhtmltopdf.org/
+
+> wkhtmltopdf is open source (LGPLv3) command line tool to render HTML into PDF using the Qt WebKit rendering engine.
+> This run entirely "headless" and do not require a display or display service.
+
+An extension for TAO allow to create test booklets: [taoBooklet](https://github.com/oat-sa/extension-tao-booklet).
+However this extension needs a third-party tool to generate the PDF files.
+So to be able to generate the booklet, you should install `wkhtmltopdf` on your server.
+
```

Comment:
You could mention, that there is also a binary for Windows on https://wkhtmltopdf.org/downloads.html, but that this has not been tested by us.